packages/valaxy/node/cli/new.ts

```diff
@@ -0,0 +1,39 @@
+import type { Argv } from 'yargs'
+
+export const newPost = (cli: Argv<{}>) => {
+  cli.command(
+    'new <title>',
```

Comment:
We can use `n` (alias) for new.

Reply:
Personally, I do not think adding an alias for sub command is a good idea, since the user can do that themselves.

For example, in **zsh** we have `gco` for `git checkout`, but that was not added by `git-checkout`.

Reply:
OK. But I see that nestJS has this alias `n` for `new`.
Of course, we can think about it later.